apps/src/studio/StudioAnimation.js

```diff
@@ -84,13 +95,32 @@ StudioAnimation.prototype.getElement = function () {
 
 /**
  * Create an image element with a clip path
+ * (if the element and clip path already exist, update the dimensions and URL)
  */
 StudioAnimation.prototype.createElement = function (parentElement) {
```

Comment:
Fixed by removing this code. We now only call `createElement` when we actually want to create the element.